### PR TITLE
[WIP] Patch vb script

### PR DESCRIPTION
**Pull request by Jaybsoni**
### Before submitting

Please complete the following checklist when submitting a PR:

- [ ] All new features must include a unit test.
      If you've fixed a bug or added code that should be tested, add a test to the
      [`tests`](../tests) directory!

- [ ] All new functions and code must be clearly commented and documented.
      If you do make documentation changes, make sure that the docs build and
      render correctly by running `make docs`.

- [ ] Ensure that the test suite passes, by running `make test`.

- [ ] Add a new entry to the `.github/CHANGELOG.md` file, summarizing the
      change, and including a link back to the PR.

- [ ] Ensure that code is properly formatted by running `make format`. 

When all the above are checked, delete everything above the dashed
line and fill in the pull request template.

------------------------------------------------------------------------------------------------------------

**Context:**
Patching the automated vb script to avoid some of the code-factor messages that come up on the PR and reduce the efficiency of using automated version bumps

### CODE REVIEW

File: .github/workflows/post_release_version_bump.yml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 name: Automated Release Version Bumps
 
 on:
+  workflow_dispatch:
```
Comment (Jaybsoni) [inline .github/workflows/post_release_version_bump.yml:4]:
```suggestion
    push:
    branches:
      - patch_vb_script
      
```

File: .github/workflows/post_release_version_bump.yml [hunk 2]
```diff
@@ -1,6 +1,10 @@
 name: Automated Release Version Bumps
 
 on:
+    push:
+    branches:
+      - patch_vb_script
+      
   release:
     types: [published]
```
Comment (Jaybsoni) [inline .github/workflows/post_release_version_bump.yml:9]:
```suggestion
    push:
    branches:
      - patch_vb_script
#  release:
#    types: [published]
```

File: .github/workflows/post_release_version_bump.yml [hunk 3]
```diff
@@ -1,8 +1,11 @@
 name: Automated Release Version Bumps
 
 on:
-  release:
-    types: [published]
+    push:
+    branches:
+      - patch_vb_script
+#  release:
+#    types: [published]
```
Comment (Jaybsoni) [inline .github/workflows/post_release_version_bump.yml:8]:
```suggestion
    push:
        branches:
            - patch_vb_script
```

File: .github/workflows/post_release_version_bump.yml [hunk 4]
```diff
@@ -1,8 +1,9 @@
 name: Automated Release Version Bumps
 
 on:
-  release:
-    types: [published]
+    push:
+        branches:
+            - patch_vb_script
```
Comment (Jaybsoni) [inline .github/workflows/post_release_version_bump.yml:6]: Done testing the patch
```suggestion
    release:
        types: [published]
```

File: .github/workflows/post_release_version_bump.yml [hunk 5]
```diff
@@ -1,8 +1,8 @@
 name: Automated Release Version Bumps
 
 on:
-  release:
-    types: [published]
+    release:
+        types: [published]
```
Comment (Jaybsoni) [inline .github/workflows/post_release_version_bump.yml:5]:
```suggestion
   release:
      types: [published]
```

File: .github/workflows/post_release_version_bump.yml [hunk 6]
```diff
@@ -1,8 +1,8 @@
 name: Automated Release Version Bumps
 
 on:
-  release:
-    types: [published]
+   release:
+      types: [published]
```
Comment (Jaybsoni) [inline .github/workflows/post_release_version_bump.yml:5]:
```suggestion
  release:
    types: [published]
```

**Review by antalszava — APPROVED** [review]
:ok_hand: Nice :)

**Pull request merged**